src/plugin/analytics/Resources/modules/administration/dashboard/components/tool.jsx

```diff
@@ -49,6 +50,15 @@ const DashboardTool = (props) =>
 
           return Community
         }
+      }, {
+        path: '/logs',
+        render: () => {
+          const Log = (
+            <DashboardLog count={props.count} />
+          )
+
+          return Log
```

Comment:
You don't need to pass the `count` prop, you don't use it.
Also, the `render()` method on routes is useful to forward props from the parent to routed components, but when you don't need it, it is better (more efficient) to write : 

```
{
  path: '/logs',
  component: DashboardLog
}
```

Reply:
Change done, thx :)